src/room/GroupCallView.tsx

```diff
@@ -293,7 +295,7 @@ export function GroupCallView({
           setUserChoices(choices);
           enter();
         }}
-        muteAudio={participants.size > 8}
+        initWithMutedAudio={participants.size > 0}
```

Comment:
This should be > 8 ?
Or did the logic change?

Reply:
Oops